tide_core.install

```diff
@@ -945,6 +945,9 @@ function tide_core_update_8038() {
   foreach ($configs as $config) {
     $editable_config = $config_factory->getEditable($config);
     $rows = $editable_config->get('settings.toolbar.rows.0');
+    if (!isset($rows)) {
```

Comment:
On a fresh install the config factory could not access the active configuration. Escaping early seems to be fine as the correct config is being installed for a fresh install.